src/app/components/MessageBanner/index.test.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { render, screen } from '../react-testing-library-with-providers';
+import MessageBanner from '.';
+import kyrgyzBanner from './fixtures';
+
+const { summaries } = kyrgyzBanner;
+
+describe('MessageBanner', () => {
+  it('should render a section with role region', () => {
+    render(<MessageBanner summaries={summaries} title={kyrgyzBanner.title} />);
+    const region = screen.getByRole('region');
+    expect(region).toBeInTheDocument();
+  });
+
+  it('has correct aria-labelledby attribute', () => {
+    const { getByRole } = render(
+      <MessageBanner summaries={summaries} title={kyrgyzBanner.title} />,
+    );
+    const messageBannerEl = getByRole('region');
+    expect(messageBannerEl.getAttribute('aria-labelledby')).toBe(
+      'message-banner',
+    );
+  });
+
+  it('should display the banner title correctly as an H2', () => {
+    render(<MessageBanner summaries={summaries} title={kyrgyzBanner.title} />);
+    expect(screen.getByText(kyrgyzBanner.title).nodeName).toBe('H2');
+  });
+
+  it('should have a heading with id of message-banner', () => {
+    render(<MessageBanner summaries={summaries} title={kyrgyzBanner.title} />);
+    const heading = screen.getByText(kyrgyzBanner.title);
+    expect(heading.getAttribute('id')).toBe('message-banner');
+  });
```

Comment:
Just had a thought about this test - I think it would be good for us to check that the heading id matches the aria-labelledby (because that's how we tell the screenreader what the label for that region is. It means that if anyone ever changes either the `id` or the `aria-labelledby`, and the values are not the same, then the test will fail.

Reply:
Test updated to check that the id and aria-labelledby are the same